eng/versioning.targets

```diff
@@ -0,0 +1,82 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We should figure out if we should move this into the version.targets in arcade so we can share.

Reply:
I'd be in favor of that, this one is almost identical to the one in CoreFx